tst/regression/test_suites/advection_outflow/advection_outflow.py

```diff
@@ -48,7 +48,8 @@ def Analyse(self, parameters):
                 "outflow.out0.00001.phdf",
                 parameters.parthenon_path
                 + "/tst/regression/gold_standard/outflow.out0.00001.phdf",
-            ]
+            ],
+            all=True,
```

Comment:
Am I missing sth or is `all` never really used anywhere in the `compare()` function?

Reply:
Good catch, all is not used at all. I've removed it. Also, I'm renaming the new option to "check_input" since we're really checking the entire "Input" group in the HDF5 file, not just the "Input.File" attribute (but it's the only one under "Input").